src/st/np3/collision.c

```diff
@@ -0,0 +1,12 @@
+#include <common.h>
+
+extern u16 g_testCollEnemyLookup[];
+extern u8 g_testCollLuckCutoff[];
+extern u8 g_testColluCoords[];
+extern u8 g_testCollvCoords[];
+extern u16 g_testCollElementLookup[10];
+extern u8 g_testColliFrames[10];
+extern u16 g_testCollPrizeTable[];
+extern u16 g_testCollRandTable[];
```

Comment:
Same idea here.

Reply:
Thank you, good point. I forgot to clean that up while I was moving stuff. Great catch!